week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -51,13 +53,17 @@ console.log(x);
 
 const y = { x: 9 };
 
-function f2(val) {
-  val.x = val.x + 1;
+function f2(val)---x:9 {
+  val.x = val.x + 1;-
   return val;
 }
 
 f2(y);
 console.log(y);
-```
+
+will be 10 because function  f2 passing object y as parameter  val.x=9+1 10 i think
```

Comment:
Not sure what you mean here @Taslima26 - do you mean that x will be printed as `10` ?

Reply:
Yes .

Reply:
I found this one bit tricky to be honest!